packages/story-editor/src/app/highlights/states.ts

```diff
@@ -22,147 +22,141 @@ import { STYLE_PANE_IDS } from '@googleforcreators/design-system';
 /**
  * Internal dependencies
  */
-import { INSERT, DOCUMENT, STYLE } from '../../components/sidebar';
+import {
+  INSERT,
+  DOCUMENT,
+  STYLE as STYLE_SIDEBAR,
+} from '../../components/sidebar/constants';
 import { PANE_IDS } from '../../components/library/paneIds';
+import type { HighlightState } from '../../types/highlightsProvider';
 
-/**
- * Highlight state object
- *
- * @typedef {Highlight} Highlight The current state of editor highlights
- * @property {string} tab The ID of the Sidebar tab to highlight
- * @property {boolean} focus Whether there is a focusable element, use for accessibility
- */
-
-const keys = {
-  STORY_TITLE: 'STORY_TITLE',
-  ELEMENT_TOOLBAR_TOGGLE: 'ELEMENT_TOOLBAR_TOGGLE',
+export enum HighlightType {
+  STORY_TITLE = 'STORY_TITLE',
```

Comment:
The standard for enums in PascalCase (StoryTitle instead of `STORY_TITLE`), we've been following that in our codebase, too.

Reply:
Okay, it was a pain, but I converted all the enum to use PascalCase. This was 50 files changes and makes the PR much bigger.